specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/privateLinkServicesForEDMUpload.json

```diff
@@ -0,0 +1,1255 @@
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "Get all the privateLinkServicesForEDMUpload instances in a subscription.",
+        "operationId": "privateLinkServicesForEDMUpload_List",
+        "x-ms-examples": {
+          "List all services in subscription": {
+            "$ref": "./examples/EdmUploadServiceList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is a synchronous operation. The body contains a JSON-serialized array of the metadata from all the privateLinkServicesForEDMUpload instances in the subscription.",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkServicesForEDMUploadDescriptionListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityAndCompliance/privateLinkServicesForEDMUpload": {
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "Get all the service instances in a resource group.",
+        "operationId": "privateLinkServicesForEDMUpload_ListByResourceGroup",
+        "x-ms-examples": {
+          "List all services in resource group": {
+            "$ref": "./examples/EdmUploadServiceListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is a synchronous operation. The body contains a JSON-serialized array of the metadata from all the privateLinkServicesForEDMUpload instances in the resource group.",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkServicesForEDMUploadDescriptionListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/providers/Microsoft.SecurityAndCompliance/operations": {
```

Comment:
>  "/providers/Microsoft.SecurityAndCompliance/operations": { [](start = 2, length = 60)

this one too - this should only be declared in one place

Reply:
moved it to common-types.json